README.md

```diff
@@ -112,20 +112,14 @@ on:
     branches:
       - master
 jobs:
-  check:
+  check_dco:
     runs-on: ubuntu-latest
+    name: Check DCO
     steps:
-    - uses: actions/checkout@v3
-    - name: Set up Python 3.x
-      uses: actions/setup-python@v4
-      with:
-        python-version: '3.x'
-    - name: Check DCO
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      run: |
-        pip3 install -U dco-check
-        dco-check
+      - name: DCO Compliance Check
+        uses: christophebedard/dco-check@0.5.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I like this, but it doesn't allow you to pass CLI args like `--verbose`, e.g.: https://github.com/christophebedard/dco-check/blob/30353d8deedf393cf55ba33355e71da7fdd095c7/.github/workflows/dco.yml#L25. I believe you could just add an action input (`args`) for this and provide it when calling `dco_check.py` in `action.yml`: https://docs.github.com/en/actions/sharing-automations/creating-actions/metadata-syntax-for-github-actions#inputs.

Also, I'd like it to be tested in GitHub CI. You could replace this part with the new action: https://github.com/christophebedard/dco-check/blob/30353d8deedf393cf55ba33355e71da7fdd095c7/.github/workflows/dco.yml#L16-L25. Also, please make the Python version an action input too (`python-version`, just like `actions/setup-python`), with default being `3.12`, this way it can be parametrized like in `dco.yml`.